### PR TITLE
Groovy dev cob4

### PR DESCRIPTION
**Pull request by fmessmer**
This pull request supersedes ipa-nhg's pull request #142, i.e. #142 should be empty after accepting this one.

It finishes the restructuring of the cob stacks started because of the integration of cob4 urdfs

This pull request is related to pull request pull request 70 in cob_common

### CODE REVIEW

**Comment by floweisshardt** [comment]
hard to oversee, but visual review: OK

File: cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-5.launch [hunk 1]
```diff
@@ -81,15 +97,19 @@
 	</node>
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="pc1 CPU Usage"/>
+		<prame name="topic_name" value="/pc1_usage"/>
```
Comment (NadiaHG) [inline cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-5.launch]: type error : param name="topic_name" value="/pc1_usage"/

File: cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-5.launch [hunk 2]
```diff
@@ -81,15 +97,19 @@
...
 	</node>
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="pc2 CPU Usage"/>
+		<prame name="topic_name" value="/pc2_usage"/>
```
Comment (NadiaHG) [inline cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-5.launch]: type error : param name="topic_name" value="/pc2_usage"/

File: cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-5.launch [hunk 3]
```diff
@@ -81,15 +97,19 @@
...
 	</node>
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="pc3 CPU Usage"/>
+		<prame name="topic_name" value="/pc3_usage"/>
```
Comment (NadiaHG) [inline cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-5.launch]: type error : param name="topic_name" value="/pc3_usage"/

File: cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-5.launch [hunk 4]
```diff
@@ -81,15 +97,19 @@
...
 	</node>
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="Wifi Status (ddwrt)"/>
+		<prame name="topic_name" value="/wifi_status"/>
```
Comment (NadiaHG) [inline cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-5.launch]: type error : param name="topic_name" value="/wifi_status"/

**Comment by NadiaHG** [comment]
tested on cob3-3: OK

**Pull request merged**